util/dvsim/SimCfg.py

```diff
@@ -504,6 +507,12 @@ def _create_deploy_objects(self):
         self.runs = ([]
                      if self.build_only else self._expand_run_list(build_map))
 
+        # In GUI mode, only allow one test to run.
+        if self.gui and len(self.runs) > 1:
+            self.runs = self.runs[:1]
+            log.error("In GUI mode, only one test is allowed to run. "
+                      "Picking {}".format(self.runs[0].full_name))
```

Comment:
I think this should either be `log.error` and exit immediately with a failure code or `log.warning`.

Reply:
Updated.